backend/routes/files.routes.js

```diff
@@ -0,0 +1,32 @@
+const router = require('express').Router()
+const FilesController = require('../controller/files.controller')
+const { authenticateUser } = require('../auth/middleware')
+const multer = require('multer')
+const storage = multer.memoryStorage()
+const upload = multer({ storage: storage })
+router.get('/:id', authenticateUser, FilesController.getFileController)
+router.get(
+    '/getallbyreference/:reference',
+    authenticateUser,
+    FilesController.getAllFilesByReferenceController
+)
+router.post(
+    '/',
+    authenticateUser,
+    upload.single('file'),
+    FilesController.createFileController
+)
+router.post(
+    '/bulk/:reference',
+    authenticateUser,
+    upload.array('files'),
+    FilesController.bulkCreateFileController
+)
+router.delete(
+    '/bulk',
+    authenticateUser,
+    FilesController.bulkDeleteFileController
+)
```

Comment:
why is the pattern for this different from the bulk post? if we're not using this yet, we can remove it now and add it later.